app/src/main/java/org/jellyfin/androidtv/util/KeyProcessor.java

```diff
@@ -95,15 +95,14 @@ public static boolean HandleKey(int key, BaseRowItem rowItem, Activity activity)
                                 return true;
                             case Series:
                             case Season:
-                            case BoxSet:
-                                createPlayMenu(rowItem.getBaseItem(), true, false, activity);
+                                // play next, resume or play
+                                PlaybackHelper.playOrPlayNextUp(item, activity);
```

Comment:
I think we should keep the menu for series/seasons/playlists

Reply:
I disagree, if the user wants a menu there is the menu key.
On every music player ever, if you use a explicit "play" action on a playlist it will start play and not show a menu.
On series/seasons the only other meaningful play related action is "play all", which is useless since we also have a queue follow-up settings.
If the user hit "play" on a season/series its pretty clear what the user wants to happen and that's what the new play code does.

We should never show a menu on a "play" action, since we have a "menu" key for it. Play should always default to a actual play action. As noted thats also how the web interface works, if i hit the "play" icon there it plays.